_meeting-notes/10-10-2019.md

```diff
@@ -0,0 +1,27 @@
+# Sync Oct 10, 2019
+
+## Attendees
+
+- David Dias
+- Jim Pick
+
+## Notes
+
+- AI(David): create PR to list the current team and roles 
+  - Raul -> Architect 
+  - Jim
+  - David
+- AI(David): Review the 1st video demo by Jim
+- AI(Jim): Open issues that cover the following topics/questions to get answers/shared understanding
+  - 1) What are the main differences (scope, features, ) between TestGround, TestLab, ipfs/benchmarks?
```

Comment:
Started an issue for Testlab: https://github.com/ipfs/testground/issues/64

Reply:
And here's the write-up for ipfs/benchmarks: https://github.com/ipfs/testground/issues/66